api-interop-layer/data/forecast/index.js

```diff
@@ -45,18 +60,43 @@ export default async ({ grid }) => {
     })
     .filter(({ time }) => time.isAfter(earliest));
 
-  h.forEach((hour) => {
+  // Do unit conversions on all the hourly properties. Each item in the array
+  // is an object representing one hour. Each property in the object represents
+  // a measurable value.
+  orderedHours.forEach((hour) => {
```

Comment:
can we do an eta reduction? `orderedHours.forEach(convertProperties)`? if this isn't clearer than what we have already, feel free to disregard.